.gitignore

```diff
@@ -261,3 +261,4 @@ paket-files/
 **/.vscode/**
 # C/C++ extension for VS Code
 .browse.VC.db*
+/out/build/x64-Debug
```

Comment:
In another PR you used just /out/build.
Maybe this one should be the same for uniformity?